Assets/Scripts/Minigames/PaperPlanes/MovementPlane.cs

```diff
@@ -0,0 +1,58 @@
+using UnityEngine;
+
+namespace Minigames.PaperPlanes
+{
+    public class MovementPlane : MonoBehaviour
+    {
+        // Editable in Unity Editor
+        public float Speed = 1f;
+        public float Rotate = 0.1f;
+
+        private float _randomTime = 5;
+        private bool _leftright;
```

Comment:
mischien de naam aanpassen naar iets als RotatingLeft of een comment die aangeeft welke richting true is